tests/ci/run_benchmark_build_tests.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -ex
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+source tests/ci/common_posix_setup.sh
+
+install_dir="$(pwd)/../librypto_install_dir"
+openssl_url='https://github.com/openssl/openssl.git'
+openssl_1_1_branch='OpenSSL_1_1_1-stable'
+openssl_3_0_branch='openssl-3.0'
```

Comment:
Small feature creep: can we add in a build with BoringSSL?

Reply:
BoringSSL isn't currently working with our benchmarking tools :(

Reply:
It's one of the todos to fix within the SIM :)